### ISSUE TITLE
New babel setup breaks on some features still

### ISSUE THREAD

**Issue by tannerlinsley** [nozzle/react-static#158]
cc: @EmilTholin 
I get this error when using async named exports:
```
Property declaration of ExportNamedDeclaration expected node to be of a type ["Declaration"] but instead got "ExpressionStatement"
```

Looks like we need to wait for this to merge: https://github.com/facebook/regenerator/pull/319

**Comment by EmilTholin** [comment]
Shoot, good find. Is it worth a revert-and-wait on https://github.com/nozzle/react-static/pull/134?

**Comment by tannerlinsley** [comment]
Possibly. It's hard to gauge. What do you think?

**Comment by tannerlinsley** [comment]
I think it's worth a revert, but I want to keep the changes you made to the index.js file. Possible?

**Comment by EmilTholin** [comment]
Yeah, it's probably for the best. That PR has not been merges for a month either, so it may take a while.

The changes to `index.js` aren't really doing anything without `esm` and `babel 7`. It might be best to wait with that as well?

**Comment by EmilTholin** [comment]
If I was a horrible person, I would use this opportunity to chill for `fast-async` one last time. 🤓

Seriously though. I think it's best to revert it, create a new PR and have that waiting until that PR you linked is merged and released with a new version of the Babel 7 beta.

**Comment by tannerlinsley** [comment]
Good plan. I'll do that now.

**Comment by tannerlinsley** [comment]
K, it's reverted, if you would like to open up the PR again, that would be great.

**Comment by tannerlinsley** [comment]
And since this is reverted, let's track the issue in the newly created PR, a copy and paste of the description should do.
